### ISSUE TITLE
Be able to set working directory or use the project path as default working directory

### ISSUE THREAD

**Issue by leandrorodrigueszup** [ZupIT/horusec#815]
**What would you like to be added**:
As far as I can tell, horusec assumes that the current directory as the `working dir`. It would be useful if horusec supports a different `working dir` or if treats the `project path` as so.

Actual usage.

```bash
$ cd `project diretory` (where my horusec-config.json is in)
$ ./horusec start -e -p ./
```
If I don't do the first step, horusec does not find `horusec-config.json` file. Would be very nice if it does.

I could pass the working dir:

```bash
$ ./horusec start -e -p ./ -W ./
```
Or use the project path as working dir. (maybe it could be a brocking change)

**Why is this needed**:

Basically, I wanted to create a `gradle task` the wrap the horusec binary. I couldn't make it work because of this behave.

**Comment by matheusalcantarazup** [comment]
Hi @leandrorodrigueszup thanks for the suggestion. 

Use the `--config-file-path` would not help you in this case? 

`./horusec start -e -p ./ --config-file-path path/to/your/config-file`

**Comment by leandrorodrigueszup** [comment]
Hi @matheusalcantarazup thanks for the reply!

Yes,  it works! :-) 
But still a little verbose and unnatural in my opinion.

But anyways, I can move on now. Thanks!

**Comment by matheusalcantarazup** [comment]
This is good! I'll let this issue open for further discussion.

Thank very much for your suggestion.
